Visualizer/src/components/dashboard/AgentInfo.vue

```diff
@@ -251,7 +391,12 @@ const allocations = computed(() => {
       },
       {
         label: "Collisions",
-        value: stat.collidingAgentBids.length,
+        value: Object.keys(stat.collidingAgentBids).length,
+        icon: GitPullRequest,
+      },
+      {
+        label: "Displacements",
+        value: Object.keys(stat.displacingAgentBids).length,
```

Comment:
displacingAgentBids are the bids of the agent that forced the reallocation to the allocation

Reply:
What would be a good name then? 🤔

Reply:
Ditch das field, es ist entweder 1, wenn die allocation eine reallocation ist oder sonst 0